testing/endtoend/types/fork.go

```diff
@@ -45,7 +56,27 @@ func InitForkCfg(start, end int, c *params.BeaconChainConfig) *params.BeaconChai
 	}
 	// Time TTD to line up roughly with the bellatrix fork epoch.
 	// E2E sets EL block production rate equal to SecondsPerETH1Block to keep the math simple.
-	ttd := uint64(c.BellatrixForkEpoch) * uint64(c.SlotsPerEpoch) * c.SecondsPerSlot
+	// If Bellatrix is disabled (math.MaxUint64), the chain starts post-merge so TTD should be 0.
+	ttdEpoch := c.BellatrixForkEpoch
+	if ttdEpoch == math.MaxUint64 {
+		ttdEpoch = 0
+	}
+	ttd := uint64(ttdEpoch) * uint64(c.SlotsPerEpoch) * c.SecondsPerSlot
 	c.TerminalTotalDifficulty = fmt.Sprintf("%d", ttd)
+
+	// Update blob schedule to use the modified fork epochs.
+	// Only include entries for forks that are enabled (not set to MaxUint64).
+	c.BlobSchedule = nil
+	if c.DenebForkEpoch != math.MaxUint64 {
+		c.BlobSchedule = append(c.BlobSchedule, params.BlobScheduleEntry{
+			Epoch: c.DenebForkEpoch, MaxBlobsPerBlock: uint64(c.DeprecatedMaxBlobsPerBlock),
```

Comment:
Same question as in `config/params/testnet_e2e_config.go` - can we use something else than a deprecated field?